src/backend/core/resource_server/urls.py

```diff
@@ -0,0 +1,11 @@
+"""Resource Server URL Configuration"""
+
+from django.urls import path
+
+from .views import DataView, JWKSView
+
+urlpatterns = [
+    path("jwks", JWKSView.as_view()),
+    # FIXME: temporary route
+    path("data", DataView.as_view()),
```

Comment:
conceptually, a resource server is just an API returning data on an endpoint with specific authentication? So could we just served groups of the existing /groups endpoint and "just" add a new authentication backend to it?
Or are we receving the request with a "scope" argument and we must know the possible scopes in advance to and return them all from a unique /data url (what your views seem to do)?